components/engine_test/src/lib.rs

```diff
@@ -115,31 +116,33 @@ pub mod kv {
     pub struct TestTabletFactory {
         root_path: PathBuf,
         db_opt: DbOptions,
-        cf_opts: Vec<(&'static str, KvTestCfOptions)>,
         root_db: Arc<Mutex<Option<KvTestEngine>>>,
     }
 
     impl TestTabletFactory {
-        pub fn new(
-            root_path: &Path,
-            db_opt: DbOptions,
-            cf_opts: Vec<(&'static str, KvTestCfOptions)>,
-        ) -> Self {
+        pub fn new(root_path: &Path, db_opt: DbOptions) -> Self {
```

Comment:
Why change?

Reply:
When it comes to multi-rocksdb, cf_opts is bound to individual rocksdb, instead of tablet factory.